Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerExchangeInformation.ps1

```diff
@@ -68,15 +68,15 @@ Function Invoke-AnalyzerExchangeInformation {
     }
 
     if ($null -ne $exchangeInformation.BuildInformation.KBsInstalled) {
-        $AnalyzeResults | Add-AnalyzedResultInformation -Details ("Exchange IU or Security Hotfix Detected.") `
+        $AnalyzeResults | Add-AnalyzedResultInformation -Name ("Exchange IU or Security Hotfix Detected.") `
```

Comment:
Nit: Can you please remove the point ("."). We automatically add a double point because we pass it through the Name parameter.